CHANGELOG.md

```diff
@@ -4,5 +4,5 @@ The following is a curated list of changes in the Enact docs-utils module, newes
 
 ## [0.4.0] - 2022-10-20
 
-- Update dependency with documentation 14.0.0
+- Update dependency with documentation 14.0.0 (require node 14 or later)
```

Comment:
Please tell me why you changed the existing change log already released.
This is a very bad example. We should not modify released change logs.
Please write a new one for this change.